Update checklist: remove new core devs from triage team (#1191)
python/devguide: core-developers/become-core-developer.rst

#. A core developer (submitter, usually the mentor) starts a poll in the
   `Committers category`_ on the `Python Discourse`_.

>    - Open for 7 days.
>    - Results shown upon close.

Author: @CAM-Gerlach Is this better?
```suggestion
   - open for 7 days
   - results shown upon close
```

   (approver) will email the candidate:

   - A request for account details as required by
     `🔒 python/voters <https://github.com/python/voters>`_.
   - A reminder about the `Code of Conduct`_ and guidance on reporting issues
     to the PSF Conduct WG.

#. Once the candidate has provided the pertinent details, the approver will:

   - Enable the various new privileges.

>    - Remove from triage team, if applicable.
>    - Add the new committer's details to
>      `🔒 python/voters <https://github.com/python/voters>`_.
>    - Update the devguide to publicly list the new committer's team membership
>      at :ref:`developers`.

Reviewer: ```suggestion
   - Remove the new committer from the triage team, if applicable.
   - Add their details to `🔒 python/voters <https://github.com/python/voters>`_.
   - Update the devguide to publicly list their team membership
     at :ref:`developers`.
```
